media_driver/linux/common/codec/ddi/media_ddi_decode_vp8.cpp

```diff
@@ -235,14 +235,13 @@ VAStatus DdiDecodeVP8::ParsePicParams(
     codecPicParams->ucUvModeProbs[0]            = picParam->uv_mode_probs[0];
     codecPicParams->ucUvModeProbs[1]            = picParam->uv_mode_probs[1];
     codecPicParams->ucUvModeProbs[2]            = picParam->uv_mode_probs[2];
-    if (codecPicParams->ucMvUpdateProb[0] && picParam->mv_probs[0])
```

Comment:
what was the particular compilation warning here and why did you drop these checks?

Reply:
> what was the particular compilation warning here and why did you drop these checks?

It is [-Werror=address] .
codecPicParams->ucMvUpdateProb and picParam->mv_probs are array 2 dimensi as below.
    _uint8_t             ucMvUpdateProb[2][CODEC_VP8_MVP_COUNT];      // 2 for decoding row and column components
    uint8_t mv_probs[2][19];_
codecPicParams->ucMvUpdateProb[0], picParam->mv_probs[0], codecPicParams->ucMvUpdateProb[1] and picParam->mv_probs[1] could never be nullptr.

Reply:
ok, thx.